xds/src/test/java/io/grpc/xds/sds/TlsCertificateSecretVolumeSecretProviderTest.java

```diff
@@ -95,6 +95,11 @@ public void verifyCallbackExecuted()
       public void updateSecret(TlsCertificateStore secret) {
         listenerRun = true;
       }
+
+      @Override
+      public void onException(Throwable throwable) {
+
```

Comment:
nit: empty line is not allowed. should this throw exception since the test doesn't expect to receive this callback? also 1 other place.

Reply:
@creamsoup the TlsCertificate* classes/code are going to be removed in a later PR. We will only be using SslContext* classes from the SecretManager. I think it is not worthwhile to fix these